.github/workflows/ci.yml

```
@@ -182,19 +182,30 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
 
-  build-test-gcp-image:
+  build-test-gcp-sev-image:
     strategy:
       matrix:
         distrib: [ubuntu, rhel]
     name: GCP image (${{ matrix.distrib }})
```

If you don't mind (clearer on github pipeline):

```suggestion
    name: ${{ matrix.distrib }} -> GCP image
```